examples/api/service/readme.md

```diff
@@ -66,7 +68,7 @@ Next, we are going to build our service.
 		}
 		defer con.Close()
 
-		fmt.Fprintln(con, "Hello! This is whyrusleepings awesome ipfs service")
+		fmt.Fprintln(con, "ZERO IPFS service. Nothing to see here.")
```

Comment:
(for what it's worth, my own opinion that should not be counted here is that
```
"Hello! This is whyrusleepings awesome ipfs service"
```
is light-hearted and friendly, which is always nice if the user is struggling with the code.